### PR TITLE
adding marketing slugs

### PR DESCRIPTION
**Pull request by ehfeng**

### CODE REVIEW

File: src/sentry/constants.py [hunk 1]
```diff
@@ -263,13 +263,32 @@ def _load_platform_data():
...
 # special cases where the marketing slug differs from the integration id
 # (in _platforms.json). missing values (for example: "java") should assume
-# the marketing slug is the same as the integration id.
+# the marketing slug is the same as the integration id:
+# javascript, node, python, php, ruby, go, swift, objc, java, perl, elixir
 MARKETING_SLUG_TO_INTEGRATION_ID = {
-    "kotlin": "java",
-    "scala": "java",
+    "kotlin": "java-android",
+    "scala": "java-logback",
```
Comment (bretthoerner) [inline src/sentry/constants.py]: I'd leave these as `java`. Kotlin is also used on the server side, and Scala can use any of the logging integrations (or none at all) so the main Java page is better.

**Review by bretthoerner — APPROVED** [review]
Looks good except for my comment.

File: src/sentry/constants.py (hunk 1 above)
Comment (ehfeng) [inline src/sentry/constants.py]: Thanks, done!

Would be nice to eventually have docs for Kotlin, even if they're basically, "it just works with Java stuff."

**Comment by bretthoerner** [comment]
Ya both Scala and Kotlin could probably use a page that says that. BUT I think 99% of users of those languages are very used to using/looking for Java libs.

**Comment by ehfeng** [comment]
Ah good point.

**Pull request merged**